### DISCUSSION TITLE
前台悬浮窗  这个开关是什么意思啊

### DISCUSSION THREAD

**Discussion by shuohao** [gkd-kit/gkd#530] [Q&A] [answered]
我开启了之后，界面上也没有看见悬浮窗

**Comment by lisonge** [comment, marked as answer]
因为它是透明的，本质是告诉系统我占着前台，提高运行优先级

在我的手机上如果关闭的话会导致响应变慢

此外我在最开始开发的时候，如果没有开启通知栏和透明悬浮窗，仅开启无障碍服务

此时当应用处于后台时，在无障碍服务里启动的子线程或协程不被系统执行

**Reply by shuohao** [reply]
确实，我刚关闭之后 直接开屏广告都不跳过了。。。 小米12pro 澎湃os
